### ISSUE TITLE
Bug When item is added to series one file overtakes the series and only shows that one

### ISSUE THREAD

**Issue by ThePromidius** [Kareadita/Kavita#1364]
(I'm bad at making titles change the issue title as you want)
**Describe the bug**
Posting as suggested by robbie on discord

TLDR:
User asks in support because:
1. He adds one single file cbr to its series folder inside a volume subfolder
2. Scans the series.
3. Only one different volume is shown
Expected behaviour:
All volumes should be shown.

**To Reproduce**

1. Replicate the following library:
```\---Nightwing
    +---Nightwing Vol 2
    |       Nightwing_001.cbz
    |              Metadata:  (shortened)
    |                       <Series>Nightwing</Series>
    |                       <Number>1</Number>
    |                       <Volume>1996</Volume>
    |
    \---Nightwing Vol 3
            Nightwing_030.cbz
                   Metadata:  (shortened)
                            <Series>Nightwing</Series>
                            <Number>30</Number>
                            <Volume>2011</Volume>
```

5. Scan library to detect new serie `Nightwing`
2 volumes available:
![image](https://user-images.githubusercontent.com/34222883/177422059-8f5990d3-221f-42dd-b84e-857db9b8feb4.png)

6. Add file:
```
\---Nightwing
    \---Nightwing Vol 4
            Nightwing_093.cbr
                  Metadata:  (shortened)
                            <Series>Nightwing</Series>
                            <Number>93</Number>
                            <Volume>2016</Volume>
```
7. Scan series
8. Volume 3 / volume 2011 overtook everything:
![image](https://user-images.githubusercontent.com/34222883/177422178-7448c2ed-1890-41a4-8658-62ef571bfae4.png)

**Expected behavior**
I believe the issue is with the volume subfolders:
```
\---Nightwing
    +---Nightwing_001.cbz
    +---Nightwing_030.cbz
    +---Nightwing_093.cbr
```
Having all files in the same series folder "Nightwing". Expected result:
![image](https://user-images.githubusercontent.com/34222883/177422514-5f3032a8-77b9-45ce-a573-a2ec0a18f559.png)

**Desktop (please complete the following information):**
 - OS: Docker
 - Browser Chrome
 - Version 0.5.4.0 (Stable)

**Additional context**
Discord support message (graphical video describing the steps screenshots): https://discord.com/channels/821879810934439936/840693141082341406/993962016115015852

Some user attached logs: (may or not be related)
```
Nightwing 2022-07-05 20:39:35,stdout,      [ScannerService] Volume cleanup code was trying to remove a volume with a file still existing on disk. File: /volume1/Kavita/Comics/DC/Night wing/Nightwing Vol 1/Nightwing #001.cbr
2022-07-05 20:39:35,stdout,      [ScannerService] Volume cleanup code was trying to remove a volume with a file still existing on disk. File: /volume1/Kavita/Comics/DC/Night wing/Nightwing Vol 1/Nightwing #001.cbr

2022-07-05 20:39:35,stdout,[41m[30mfail[39m[22m[49m: API.Services.Tasks.ScannerService[0]

2022-07-05 20:39:35,stdout,      [ScannerService] Volume cleanup code was trying to remove a volume with a file still existing on disk. File: /volume1/Kavita/Comics/DC/Night wing/Nightwing Vol 2/Nightwing #001.cbz

2022-07-05 20:39:35,stdout,[41m[30mfail[39m[22m[49m: API.Services.Tasks.ScannerService[0]

2022-07-05 20:39:35,stdout,      [ScannerService] Volume cleanup code was trying to remove a volume with a file still existing on disk. File: /volume1/Kavita/Comics/DC/Night wing/Nightwing Vol 4/Nightwing #001.cbr
```
Ask more info if needed

**Comment by majora2007** [comment]
This might be fixed in latest nightly, there was a small bug in the scan loop which could cause this.

**Comment by majora2007** [comment]
I believe this is solved, I'm going to close it since original user isn't here to confirm.
